components/resume/company/positions.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from 'react'
+
+interface PositionProps {
+    title: string;
+    startDate: string;
+    endDate?: string;
+}
+
+const Position: FC<PositionProps> = ({ title, startDate, endDate }) => {
+    return <div>
+        <strong>{title}</strong> {startDate} -- {endDate}
```

Comment:
In the future I'll do some conditional logic on endDate to show "Present" if it's missing and pull in date-fns to do date string formatting.

Reply:
Added this as a tracker item to https://github.com/jarekb84/me/issues/1